### ISSUE TITLE
TypeError: Cannot read property 'body' of undefined

### ISSUE THREAD

**Issue by MaKleSoft** [Draccoz/twc#29]
This seems to happen only for target polymer 2
```sh
/Users/martin/.nvm/versions/node/v6.7.0/lib/node_modules/twc/src/parsers/JSParser.js:30
            constructor.body,
                       ^

TypeError: Cannot read property 'body' of undefined
    at Module.JSParser (/Users/martin/.nvm/versions/node/v6.7.0/lib/node_modules/twc/src/parsers/JSParser.js:30:24)
    at Module (/Users/martin/.nvm/versions/node/v6.7.0/lib/node_modules/twc/src/PolymerModule.js:78:9)
    at DestroyableTransform._transform (/Users/martin/.nvm/versions/node/v6.7.0/lib/node_modules/twc/src/index.js:53:36)
    at DestroyableTransform.Transform._read (/Users/martin/.nvm/versions/node/v6.7.0/lib/node_modules/twc/node_modules/readable-stream/lib/_stream_transform.js:159:10)
    at DestroyableTransform.Transform._write (/Users/martin/.nvm/versions/node/v6.7.0/lib/node_modules/twc/node_modules/readable-stream/lib/_stream_transform.js:147:83)
    at doWrite (/Users/martin/.nvm/versions/node/v6.7.0/lib/node_modules/twc/node_modules/readable-stream/lib/_stream_writable.js:347:64)
    at writeOrBuffer (/Users/martin/.nvm/versions/node/v6.7.0/lib/node_modules/twc/node_modules/readable-stream/lib/_stream_writable.js:336:5)
    at DestroyableTransform.Writable.write (/Users/martin/.nvm/versions/node/v6.7.0/lib/node_modules/twc/node_modules/readable-stream/lib/_stream_writable.js:274:11)
    at PassThrough.ondata (_stream_readable.js:555:20)
    at emitOne (events.js:96:13)
```

**Comment by Draccoz** [comment]
Polymer 2 output is not yet implemented, though in Beta 2 milestone (issue here: #16).
I was able to recreate above issue without a hassle for polymer v1 target and already fixed it (together with #28). I'm writing regression tests and going to push it.

**Comment by Draccoz** [comment]
Fixed in #30 PR.
